packages/react-dom/src/events/DOMTopLevelEventTypes.js

```diff
@@ -200,6 +199,6 @@ export const mediaEventTypes: Array<TopLevelType> = [
   TOP_WAITING,
 ];
 
-export function getRawEventName(topLevelType: TopLevelType): string {
-  return topLevelType;
+export function getRawEventName(topLevelType: mixed): string {
```

Comment:
Using `DOMTopLevelEventType` would require us to force a cast to `DOMTopLevelEventType` in the event plugin. My idea is to keep things limited to this file for now.